iron-resizable-behavior.html

```diff
@@ -0,0 +1,148 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<script>
+  /**
+  `IronResizableBehavior` is a behavior that can be used in Polymer elements to
+  coordinate the flow of resize events between "resizers" (elements that control the
+  size or hidden state of their children) and "resizables" (elements that need to be
+  notified when they are resized or un-hidden by their parents in order to take
+  action on their new measurements).
+  Elements that perform measurement should add the `IronResizableBehavior` behavior to
+  their element definition and listen for the `iron-resize` event on themselves.
+  This event will be fired when they become showing after having been hidden,
+  when they are resized explicitly by another resizable, or when the window has been
+  resized.
+  Note, the `iron-resize` event is non-bubbling.
+  @group Polymer Behaviors
+  @element iron-resizable-behavior
+  @homepage github.io
+  */
+  Polymer.IronResizableBehavior = {
+    properties: {
+      _parentResizable: {
+        observer: '_parentResizableChanged'
+      }
+    },
+
+    listeners: {
+      'iron-request-resize-notifications': '_onIronRequestResizeNotifications'
+    },
+
+    created: function() {
```

Comment:
Should we add a comment that it's setting these properties here instead of in `properties` because they're used by an observer?